### PR TITLE
[12.0] [MIG] account_due_list_aging_comment

### PR DESCRIPTION
**Pull request by cor9003**
Migration to 12 account_due_list_aging_comment

### CODE REVIEW

**Comment by cor9003** [comment]
Hello, The CLA has been already signed and sent by email

File: account_due_list_aging_comment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2015-2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Payments Due list aging comment",
+    'version': '12.0.1.0.0',
+    'category': 'Generic Modules/Payment',
+    'author': 'Eficent,'
+              'Odoo Community Association (OCA)',
+    'website': 'http://www.eficent.com',
```
Comment (Nikul-Chaudhary) [inline account_due_list_aging_comment/__manifest__.py]: @cor9003 URL `https://github.com/OCA/account-payment`

**Review by tbaden — CHANGES_REQUESTED** [review]
please use the [maintainer-tools](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0) to create readme.

**Comment by cor9003** [comment]
@tbaden readme created with maintainer-tools. Thanks to inform me

**Comment by tbaden** [comment]
Thanks for the work 👍
Just the request from @Nikul-Chaudhary is still open

File: account_due_list_aging_comment/__manifest__.py (hunk 1 above)
Comment (cor9003) [reply account_due_list_aging_comment/__manifest__.py]: Done !

**Comment by cor9003** [comment]
@tbaden The request from @Nikul-Chaudhary is already done

**Review by tbaden — APPROVED** [review]
Thanks, please Squash your commits into 1

**Comment by cor9003** [comment]
squash commits into 1 done. @tbaden Is there any action I should do to move forward this PR?

**Comment by tbaden** [comment]
@Nikul-Chaudhary anymore changes or good to go?

**Comment by cor9003** [comment]
@Nikul-Chaudhary @tbaden Hello, do I have to do anything to move forward this PR ?

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]
Please, maintain commit history as explained here https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0.

To save your work, you can copy the folder and paste outside the repo. Then reset the branch to v12.0, apply what in the link says in order to put add all commit history. And then copy the previous copied folder and paste in the repo overwriting it, and commit and push force.

**Comment by cor9003** [comment]
@mreficent I do not have the branch 12.0 in my forked repository and I don´t know how add this branch 12.0 in my forked repository. I would prefer delete this pull request and start from scratch and give you a clear new pull request. Is it ok for you ?

**Comment by MiquelRForgeFlow** [comment]
ok. do as you prefer. to create branches, you can always do:

git fetch origin 12.0
git checkout -b _new_branch_ origin/12.0

to reset branches, yo only new to do "git reset --hard origin/12"

to add remotes, is with "git remote add _remote_name_ `https://github.com/`_remote_name_`/{repo}.git`"

**Pull request closed without merging**